rotkehlchen/data/importer.py

```diff
@@ -84,10 +85,7 @@ def exchange_row_to_location(entry: str) -> Location:
             'export enough data for them. Simply enter your BTC accounts and all '
             'your transactions will be auto imported directly from the chain',
         )
-
-    raise UnsupportedCSVEntry(
-        f'Unknown Exchange "{entry}" encountered during a cointracking import. Ignoring it',
-    )
+    return Location.EXTERNAL
```

Comment:
Didn't we say to also add a note entry in this case to the trade itself to note the detected location name and that we don't recognize it so we set it to external?